src/app/cli/src/visualization_message.ml

```diff
@@ -0,0 +1,11 @@
+open Core_kernel
+
+let success visualization_type filepath =
+  sprintf
+    !"Successfully wrote the visualization of the %s at location: %s."
+    visualization_type filepath
+
+let bootstrap visualization_type =
```

Comment:
It seems a bit gratuitous to have a separate file for the messages here unless you want to consume them from multiple files. from what I could tell; you only use them in one function right? These may be better as local variables